tests/integration/long/utils.py

```diff
@@ -12,7 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from __future__ import print_function
```

Comment:
I'm not sure... this one is actually used anywhere.  Regardless it's absolutely safe to remove for Python3.